installers-conda/build_installers.py

```diff
@@ -0,0 +1,402 @@
+"""
+Create Spyder installers using `constructor`.
+
+It creates a `construct.yaml` file with the needed settings
+and then runs `constructor`.
+
+Some environment variables we use:
+
+CONSTRUCTOR_TARGET_PLATFORM:
+    conda-style platform (as in `platform` in `conda info -a` output)
+CONSTRUCTOR_CONDA_EXE:
+    when the target platform is not the same as the host, constructor
+    needs a path to a conda-standalone (or micromamba) executable for
+    that platform. needs to be provided in this env var in that case!
+CONSTRUCTOR_SIGNING_CERTIFICATE:
+    Path to PFX certificate to sign the EXE installer on Windows
+"""
+
+import json
+import os
+import platform
+import re
+import sys
+import zipfile
+from argparse import ArgumentParser
+from datetime import timedelta
+from distutils.spawn import find_executable
+from functools import partial
+from logging import getLogger
+from pathlib import Path
+from ruamel.yaml import YAML
+from subprocess import check_call
+from textwrap import dedent, indent
+from time import time
```

Comment:
Please organize these imports in alphabetical order and break them in two sections: `Standard library imports` and `Third-party imports`